CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.5
+
+### Added
+
+- Add new option for environments - `included_configs` - which is
+  a list of paths. Each each path represents an individual config file
```

Comment:
A couple notes:

- Options in general apply to both environments and individual releases, so I would phrase this "Introduced a new configuration option - ..."
- "Each each" ;)
- Clarify that this option is expecting *runtime* paths, otherwise we are sure to have folks who are expecting configs from their build environment to be included in production, and that's not what this option is for.